src/layers/ServerCluster.js

```diff
@@ -176,17 +176,29 @@ class ServerCluster extends Cluster {
     }
 
     // Keep tile clusters in memory and update map if needed
-    onTileLoad = (tileId, clusters) => {
+    onTileLoad = async (tileId, clusters) => {
         this.tileClusters[tileId] = clusters
 
-        // If tile still visible after loading
-        if (this.isTileVisible(tileId)) {
+        // Check if tile is still visible after loading
+        const visibleTiles = await this.getVisibleTiles()
```

Comment:
Why use a separate `const`?

Reply:
I think it looks better than 

if ((await this.getVisibleTiles()).includes(tileId)) {